### ISSUE TITLE
Switch Cas Configuration to use local CAS server

### ISSUE THREAD

**Issue by millross** [pac4j/vertx-pac4j-demo#44]
As per other demo implementations, to achieve central logout demo for CAS, we will need to use a local CAS server. Switch the url for CAS for this, and set up CentralLogout.

The central logout aspect of this depends on pac4j/vertx-pac4j#94, but will be covered in a separate issue.

**Comment by millross** [comment]
This is on branch local-cas-server-config for now
